docs/api/structures/ipc-main-event.md

```diff
@@ -3,6 +3,3 @@
 * `frameId` Integer - The ID of the renderer frame that sent this message
 * `returnValue` any - Set this to the value to be returned in a syncronous message
 * `sender` WebContents - Returns the `webContents` that sent the message
-* `reply` Function - A function that will send an IPC message to the renderer frame that sent the original message that you are currently handling.  You should use this method to "reply" to the sent message in order to guaruntee the reply will go to the correct process and frame.
```

Comment:
I think we should leave this documented for normal `ipc` events but have a new type `IpcMainHandleableEvent` that does not have a `reply` property.

Not set on that naming btw, should probs me more like

`IpcMainEvent` and `IpcMainReplyableEvent` where the replyable event `extends` the `IpcMainEvent`